change/@ni-nimble-components-15e8f878-a870-4090-8bf6-7c45c65195ba.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Can you tell what's up with the [chromatic diff](https://www.chromatic.com/test?appId=60e89457a987cf003efc0a5b&id=63c751a7ecb4314e27172acb)? Looks like non-visible changes to the DOM structure which would presumably make sense. But the contents of the DOM (the HTML code at the bottom of the page) look cut off so I can't tell if the diff is real or just a Chromatic bug.

Reply:
The table used to be an 'auto' height but is now a fixed height (500px) so the 10k rows version is a reasonable height. That's the only Chromatic diff I see (height change shown by the green rect in the image diffs, and the 'height:500px' is the DOM diff)